modules/contact/src/actions/ContactAction.C

```diff
@@ -634,6 +723,15 @@ ContactAction::addMortarContact()
       params.set<NonlinearVariableName>("variable") = variable_name;
       params.set<bool>("use_displaced_mesh") = true;
       params.set<bool>("compute_lm_residuals") = false;
+      params.set<bool>("interpolate_normals") = getParam<bool>("interpolate_normals");
+
+      // Additional displacement residual for frictional problem
+      // The second frictional LM acts on a perpendicular direction.
+      if (is_additional_frictional_constraint)
+      {
+        MooseEnum direction("direction_1 direction_2", "direction_2");
+        params.set<MooseEnum>("direction") = direction;
+      }
```

Comment:
```suggestion
        params.set<MooseEnum>("direction") = "direction_2";
```
I'm pretty sure this should work